lib/tasks/libsass.rb

```diff
@@ -19,13 +19,13 @@
   file "lib/libsass.so" => "Makefile" do
     make_program = ENV['MAKE']
     make_program ||= case RUBY_PLATFORM
-                     when /mswin/
-                       'nmake'
+                     when /mswin|mingw/
```

Comment:
As far as I understand Ruby for Windows DevKits are based on MinGW and Visual Studio should not be used due to various incompatibilities related to the runtime C++ support. Can you try switching to make and the Unix-style build (which is also supported by libsass) and see if that fixes the segmentation fault?